src/schema.graphql

```diff
@@ -286,4 +293,10 @@ type Mutation {
   setAccountStatus(uid: ID!, status: AccountStatus!): User
 
   setLevel(uid: ID!, level: Int!): User
+
+  save2fa(secret: String!, token: String!): Boolean
+
+  delete2fa(token: String!): Boolean
+
+  validate2fa(token: String!): Boolean
```

Comment:
Does this have a side effect? If not it could be under `Query`.

Reply:
Shifted